tabpy-tools/tabpy_tools/client.py

```diff
@@ -21,7 +21,7 @@
 
 logger = logging.getLogger(__name__)
 
-_name_checker = compile('^[a-zA-Z0-9-_\ ]+$')
+_name_checker = compile(r'^[a-zA-Z0-9-_\ ]+$')
```

Comment:
Haven't stepped through but I'm very surprised that this code would have worked without being a raw string

Reply:
`\ ` is not valid regex, shouldn't it be `\s`?

Reply:
Within the brackets it loses special meaning so should match literally on a forward slash

Reply:
Hmm OK so looking more closely I think the original regex meant to use a character class of `\s` instead. I'll update on next PR